xslt/functions_params_vars.xslt

```diff
@@ -139,9 +141,6 @@
                                         <xsl:when test="effort/@in = 'days'">
                                             <xsl:value-of select="effort/min * 8"/>
                                         </xsl:when>
-                                        <xsl:when test="effort/@in = 'weeks'">
```

Comment:
It looks like this part got lost? I assume this is not on purpose for this functionality? If so, could you please put it back - all the "weeks" parts that got deleted?

Thanks :)